server/command_create.go

```diff
@@ -226,10 +229,11 @@ func (p *Plugin) runCreateCommand(args []string, extra *model.CommandArgs) (*mod
 	install.Version = digest
 
 	req := &cloud.CreateInstallationRequest{
+		Name:        install.Name,
 		OwnerID:     extra.UserId,
 		GroupID:     config.GroupID,
 		Affinity:    install.Affinity,
-		DNS:         fmt.Sprintf("%s.%s", install.Name, config.InstallationDNS),
```

Comment:
Field **DNS** is Deprecated.  Question: Is upstream not using it anymore?

Reply:
Is still there, but we need to use `DNSNames` now, same as with `model.Installation.DNS` vs `DNSRecords`.

Reply:
May be we need to pass both until we remove **DNS** field. As this is deprecated, this field still should be worked. Right?

Reply:
That's not the case here, [check the installation request code](https://github.com/mattermost/mattermost-cloud/blob/master/model/installation_request.go#L67-L75). If provided, `DNS` is only added at the beginning of `DNSNames`, so it has no use on its own. I believe the only thing we do with that field is validation, and if we do something else with that field and not with DNSNames/DNSRecords I think that should be considered a bug. (cc: @gabrieljackson since he has more context here)